src/redux/departments/actionTypes.ts

```diff
@@ -0,0 +1,10 @@
+export enum DepartmentsActionTypes {
+    SET_DEPARTMENTS = 'SET_DEPARTMENTS',
+    ADD_DEPARTMENT = 'ADD_DEPARTMENT',
+    UPDATE_DEPARTMENT = 'UPDATE_DEPARTMENT',
+    DELETE_DEPARTMENT = 'DELETE_DEPARTMENT',
+    FETCH_START = 'FETCH_START',
+    FETCH_END = 'FETCH_END',
+    FETCH_ERROR = 'FETCH_ERROR'
+
```

Comment:
same here. If you have a global loader, then such actions should not belong to DepartmentsActionTypes or others